flow360/component/simulation/models/material.py

```diff
@@ -0,0 +1,67 @@
+from typing import Literal
+
+import pydantic as pd
+
+from flow360.component.simulation.framework.base_model import Flow360BaseModel
+
+
+class MaterialBase(Flow360BaseModel):
+    # Basic properties required to define a material.
+    # For example: young's modulus, viscosity as an expression of temperature, etc.
+    ...
+    name: str = pd.Field()
+
+
+class Gas(MaterialBase):
+    # contains models of getting properites, for example US standard atmosphere model
+    name: Literal["gas"] = pd.Field(frozen=True)
+    viscosity_type: Literal["Sutherland", "constant"] = pd.field("Sutherland")
+    constant_dynamic_viscosity: Optional[ViscosityType] = pd.Field(None)
```

Comment:
Can you add 2 classes for handling `Sutherland` and `constant` respectively? Move for example constant_dynamic_viscosity inside that model.

This avoids writing validations.

```python
...
viscosity_type = SutherLand(SutherlandConstants = ...),
...
```